.gitignore

```diff
@@ -84,6 +84,9 @@ ipython_config.py
 # pyenv
 .python-version
```

Comment:
This is a very good PR, but there are no tests. This commit should have 100% coverage.